src/sftp.rs

```diff
@@ -145,6 +145,7 @@ impl Sftp {
                 write_end_buffer_size.get(),
                 options.get_max_pending_requests(),
                 auxiliary,
+                options.get_tokio_compat_file_write_limit().get(),
```

Comment:
If you took my previous suggestion, then you need to also take this:

```suggestion
                options.get_tokio_compat_file_write_limit(),
```